<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDKPerfMonClass</class>
 <widget class="QDialog" name="SDKPerfMonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDKPerfMon</string>
  </property>
  <widget class="QPushButton" name="prefTestBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>性能测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pressureTestBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>压力测试</string>
   </property>
  </widget>
  <widget class="QSlider" name="maxNumVSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>并发量</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SDKPerfMon.qrc"/>
 </resources>
 <connections/>
</ui>
